### PR TITLE
Azure exportdb

### PR DESCRIPTION
**Pull request by jrudley**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no) Yes
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no) Yes
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-03a4c0831e/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-5bd9c5cf84/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-54d5d85753/).

**Comment by catlook** [comment]
Let me know when you finished making any edits you plan to so that I can schedule the publish for the 6th :) .  Thanks, Jimmy!

**Comment by jrudley** [comment]
Think I'm good unless you can tell me how to left justify an image? :)

**Comment by catlook** [comment]
I'll look into the left justification...but are you going to apply any of the edits I suggested?  You don't have to...just want to make sure you saw them!  If you want to, commit the new version when you're ready :)

**Comment by jrudley** [comment]
I don't see any of the edits you suggested? I always do but I don't see any
comments on GitHub?

File: _posts/2017-03-06-Export-Azure-SQL-DB.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- more -->
+
+Most SQL Server users are familiar with the concept of doing a backup using SQL Server Management Studio resulting in a backup (.bak) file. With Azure SQL, backup files are not supported, but instead BACPAC files are generated. A BACPAC file contains the database schema and data. If a user wants to generate a bacpac file, they can use the Azure portal, but that becomes tedious when you are doing a lot of exports each day on random databases. After doing a few exports using the portal, I decided to write up a PowerShell script with prompts to choose a DB and export to a storage container.
+
+Clone or download my repo [Here](https://github.com/jrudley/export-AzureSqlDb) to grab the PowerShell script that lets you download Azure SQL BACPAC files. Assuming you have [Azure PowerShell](https://azure.microsoft.com/en-us/downloads/) installed, you can pass in some parameters to omit certain prompts or just run it and answer the prompts .\Export-AzureSqlDB.ps1 -statusBar. It will prompt for the following:
```
Comment (jrudley) [inline _posts/2017-03-06-Export-Azure-SQL-DB.md]: You need the period and slash. I'll include them both in the bold portion, thx

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-1334d59940/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-e28697743d/)

**Comment by catlook** [comment]
Ok, Jimmy, I don't have good advice on left justifying the images.  I found a couple of suggestions that may work, so you can try them and check the preview!  I think the first one has promise!

1) Insert this line above each image line: <div style="text-align:left" markdown="1">   and insert this line after the image line:  </div> 
2) Put the images in a table

**Comment by catlook** [comment]
Crikey!!  I wrote down today...so I was two days off on the publish :(  Will you change the date for me and I'll publish asap?  Sorry, Jimmy!!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-f55b3e723b/)

**Pull request merged**